pages.es/common/usql.md

```diff
@@ -0,0 +1,36 @@
+# usql
+
+> Interfaz CLI universal para bases de datos SQL.
+> Más información: <https://github.com/xo/usql>.
+
+- Conectarse a una base de datos específica:
+
+`usql {{sqlserver|mysql|postgres|sqlite3|...}}://{{nombre_usuario}}:{{contraseña}}@{{host}}:{{puerto}}/{{nombre_base_de_datos}}`
+
+- Ejecuta comandos desde un archivo:
+
+`usql --file={{ruta/al/query.sql}}`
```

Comment:
Should the equal signs "=" be removed to ease the task of #11952 @vitorhcl?

Reply:
It depends on the command, if it doesn't work without an "=" sign we shouldn't remove it. But most commands support arguments with a space instead of an equal sign (so checking it locally and removing it if it works is the best course of action).

Reply:
Alright